client/src/client_sync/v27/mod.rs

```diff
@@ -147,5 +150,3 @@ crate::impl_client_v17__send_to_address!();
 crate::impl_client_v17__sign_message!();
 crate::impl_client_v17__sign_raw_transaction_with_wallet!();
 crate::impl_client_v21__unload_wallet!();
-crate::impl_client_v17__wallet_create_funded_psbt!();
-crate::impl_client_v17__wallet_process_psbt!();
```

Comment:
Why these two lines removed mate, was that an accident? CI still passes because these two methods are untested.

Reply:
Yeah accidental, put back now.